ydb/src/client_topic/topicwriter/writer.rs

```diff
@@ -264,21 +274,57 @@ impl TopicWriter {
         }
 
         if !messages.is_empty() {
+            // TODO: if messages aren't sent, save for next write_loop_iteration() call?
             trace!("Sending topic message to grpc stream...");
-            task_params
-                .request_stream
-                .send(stream_write_message::FromClient {
-                    client_message: Some(ClientMessage::WriteRequest(WriteRequest {
-                        messages,
-                        codec: 1,
-                        tx: None,
-                    })),
-                })
-                .unwrap(); // TODO: HANDLE ERROR
+            return match TopicWriter::retry(retrier, || async {
```

Comment:
The retry won't work this way.
If sending a message results in an error, the topic writer must reconnect to the server before retrying.